protocol/messenger_share_urls.go

```diff
@@ -302,6 +314,77 @@ func (m *Messenger) prepareEncodedCommunityChannelData(community *communities.Co
 	return encodedData, shortKey, nil
 }
 
+func (m *Messenger) ShareTransactionURL(request *requests.TransactionShareURL) (string, error) {
```

Comment:
So to build the transaction URL, I need to know all details about it? 
But most probably all of the details about the transaction are already somewhere in our database, can we just pass the transaction hash here and status-go will collect all required data?

Reply:
> It is to request the payment

wait, so there's no transaction yet? Should we call this `transactionRequest` then?
I can simply imagine a URL to the actual transaction 🤔